Umbraco-Cloud/Upgrades/Manual-Upgrades/Manual-CMS-upgrade.md

```diff
@@ -140,7 +52,7 @@ Make sure that everything works on the local clone and that you can **run the pr
 
 Before you deploy the upgraded project to the Cloud, it's important that you check if there are any [**dependencies**](../Product-Dependencies) on the new Umbraco version.
 
-Are there dependencies for Umbraco Forms or Umbraco Courier / Deploy you need to upgrade these locally, before moving on.
+Are there dependencies for Umbraco Forms or Umbraco Deploy you need to upgrade these locally, before moving on.
```

Comment:
Shouldn't this be "If updates are available for Umbraco Forms or Umbraco Deploy then you can upgrade those locally as well, before moving on."